src/Microsoft.Azure.WebJobs.Host/Bindings/Runtime/RuntimeBindingContext.cs

```diff
@@ -1,54 +0,0 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the MIT License. See License.txt in the project root for license information.
-
-using System;
-using System.Collections.Generic;
-using System.Collections.ObjectModel;
-using System.Linq;
-
-namespace Microsoft.Azure.WebJobs.Host.Bindings.Runtime
-{
-    /// <summary>
-    /// Context object for <see cref="IBinderEx"/> used to specify
-    /// binding details.
-    /// </summary>
-    public class RuntimeBindingContext
```

Comment:
When I introduced `IBinderEx`, I added this context object in an attempt to make the interface future proof. Now that I've moved to a `Binder` class, we have more flexibility in the future.